habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/CleanHabushuMojo.java

```diff
@@ -110,7 +110,8 @@ private void clean() throws MojoExecutionException {
         if (deleteVirtualEnv) {
             try {
                 PyenvAndPoetrySetup configureTools = new PyenvAndPoetrySetup(pythonVersion, usePyenv,
-                        patchInstallScript, workingDirectory, rewriteLocalPathDepsInArchives, getLog());
+                        patchInstallScript, workingDirectory, rewriteLocalPathDepsInArchives,
+                        null, null, null, null, getLog());
```

Comment:
A: Recommend overloading the constructor so we don't have pass in a bunch of `nulls`. Also gives us the ability in the javadocs to specify that without the credentials/server info, authentication capabilities will be non-existent/limited.

Reply:
Done